src/dndbeyond_dice.pyj

```diff
@@ -188,6 +188,7 @@ class DNDBRoll(Beyond20BaseRoll):
                 self._total += part.roll()
             else:
                 self._total += part
+        self._total = Math.round((self._total + Number.EPSILON) * 100) / 100

```

Comment:
Why are you adding epsilon here?

Reply:
I actually made a note of this in the commit summary.
This is a rounding trick to ensure that edge cases are rounded correctly.
https://stackoverflow.com/questions/11832914/round-to-at-most-2-decimal-places-only-if-necessary

It's probably not necessary in this case.